<launch>
    <node pkg="oldx" type="oldx_serial" name="oldx_serial" output="screen">
        <param name="port_name" type="string" value="/dev/ttySAC0" />
        <param name="baudrate" type="int" value="115200" />        
        <param name="x_min" type="double" value="-3.0" />
        <param name="x_max" type="double" value="3.0" />
        <param name="y_min" type="double" value="-3.0" />
        <param name="y_max" type="double" value="3.0" />
        <param name="z_min" type="double" value="-3.0" />
        <param name="z_max" type="double" value="3.0" />
        <param name="yaw_min" type="double" value="-120" />
        <param name="yaw_max" type="double" value="120" />
    </node>
</launch>